src/components/permissions/rowLevelPolicyHelpers.ts

```diff
@@ -31,9 +31,10 @@ export const baseJWT = { aud: 'postgraphile' }
   }
 */
 const compileJWT = (JWTelements: any) => {
-  const { userId, orgId, username, templatePermissionRows, sessionId, isAdmin } = JWTelements
+  const { userId, orgId, username, templatePermissionRows, sessionId, isAdmin, isManager } =
+    JWTelements
 
-  let JWT: any = { ...baseJWT, userId, orgId, username, sessionId, isAdmin }
+  let JWT: any = { ...baseJWT, userId, orgId, username, sessionId, isAdmin, isManager }
```

Comment:
We weren't even putting `isManager` in the JWT. We weren't actually using it anywhere, which is why we hadn't noticed the problem. But now we need to check it for look-up table authentication.